Simplified/LibraryFinder/UI/NYPLLibraryFinderViewController.swift

```diff
@@ -0,0 +1,257 @@
+      return 0
+    }
+    switch sectionType {
+    case .searchBar:
+      return 0
+    case .myLibrary:
+      return isMyLibraryHidden ? 0 : dataProvider.userAccounts.count
+    case .newLibrary:
+      return dataProvider.newLibraryAccounts.count
+    }
+  }
+  
+  override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: NYPLLibraryFinderLibraryCell.reuseIdentifier, for: indexPath) as? NYPLLibraryFinderLibraryCell else {
+      return UICollectionViewCell()
+    }
+    if indexPath.section == NYPLLibraryFinderSection.myLibrary.rawValue {
+      let account = dataProvider.userAccounts[indexPath.item]
+      cell.configureCell(type: .myLibrary, account: account)
+    } else if indexPath.section == NYPLLibraryFinderSection.newLibrary.rawValue {
+      let account = dataProvider.newLibraryAccounts[indexPath.item]
+      cell.configureCell(type: .newLibrary, account: account)
+    }
+    return cell
+  }
+  
+  // MARK: - FlowLayout
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+    return section == NYPLLibraryFinderSection.myLibrary.rawValue ? 0 : 8
+  }
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+    if section == NYPLLibraryFinderSection.newLibrary.rawValue && dataProvider.newLibraryAccounts.count == 0 {
+      return CGSize.zero
+    }
+    return CGSize(width: collectionView.bounds.width - NYPLLibraryFinderConfiguration.collectionViewContentInset * 2, height: 50)
+  }
+  
+  func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForFooterInSection section: Int) -> CGSize {
+    var height: CGFloat = 0.0
+    if section == NYPLLibraryFinderSection.myLibrary.rawValue {
+      height = 10.0
+    }
+    return CGSize(width: collectionView.bounds.width - NYPLLibraryFinderConfiguration.collectionViewContentInset * 2, height: height)
+  }
+  
+  override func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+    if indexPath.section == NYPLLibraryFinderSection.searchBar.rawValue {
+      let view = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: searchBarReuseIdentifier, for: indexPath)
+      view.addSubview(searchBar)
+      searchBar.autoPinEdgesToSuperviewEdges(with: UIEdgeInsets(top: 0, left: 0, bottom: 10, right: 0))
+      return view
+    }
+    
+    if kind == UICollectionView.elementKindSectionHeader,
+      let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind,
```

Comment:
very minor, and mostly subjective so feel free to ignore or disagree: Sometimes Xcode indents code in strange ways, once it breaks to the next line, and it becomes less readable. I was wondering if we can fix that by moving `ofKind` or perhaps `.dequeueReusableSupplementaryView` to the next line? I am still struggling with this myself about what's the best/most readable way to format long lines, honestly. Maybe @singhraman4282 has some ideas.